x/stakeibc/module.go

```diff
@@ -174,9 +174,7 @@ func (am AppModule) ExportGenesis(ctx sdk.Context, cdc codec.JSONCodec) json.Raw
 func (AppModule) ConsensusVersion() uint64 { return 1 }
 
 // BeginBlock executes all ABCI BeginBlock logic respective to the capability module.
-func (am AppModule) BeginBlock(ctx sdk.Context, _ abci.RequestBeginBlock) {
-	BeginBlocker(ctx, am.keeper, am.bankKeeper, am.accountKeeper)
-}
+func (am AppModule) BeginBlock(ctx sdk.Context, _ abci.RequestBeginBlock) {}
```

Comment:
Can you explain this change? Seems unrelated to SBU-03

Reply:
I also don't see it in the audit report - they might have changed it? Pretty sure this was flagged 🤔 the code path is unused, but I can revert this change if it makes more sense

Reply:
The current audit report is the newest version, which is updated. The old one is in [audit channel](https://slack.com/archives/#C03MRQZV31S) as a pdf if you wanna diff [](edited-with-Axolo)

Reply:
Can't find it in the older audit report, I'll just revert this